ui/narrative/methods/view_proteinstructures/spec.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name" : "View ProteinStructures",
+  "ver" : "1.0.0",
+  "authors" : [ ],
+  "contact" : "http://kbase.us/contact-us/",
+  "visble" : true,
+  "categories" : ["active", "viewers"],
+  "app_type" : "viewer",
+  "widgets" : {
+    "input" : null,
+    "output" : "KBaseProteinStructuresView"
```

Comment:
Note that this widget doesn't exist in the Narrative yet. Still fine to merge, though.

Reply:
Good.  I am developing the widget using the local narrative installation.  For now I need the NarrativeViewers and narrative_method_specs_ci to help me with the development.

Reply:
No, that doesn't have any effect.